### DISCUSSION TITLE
HeatMap

### DISCUSSION THREAD

**Discussion by Rmfjd1978** [h2oai/wave#1604] [Q&A]
I've maded an heatmap an it looks like this:
![image](https://user-images.githubusercontent.com/98091225/185904761-babef2c5-ca69-4ce5-ae28-dbf0a4a64fff.png)

Is it possible to make the heatmap ignore half like this:
![image](https://user-images.githubusercontent.com/98091225/185904953-8f850d6d-87c6-46b5-9e65-a5612e18e848.png)

another question how can i hide the label that appears in the first picture that contains 0.1088931012960?

**Comment by mtanco** [comment]
Hello! You can set the value of that data point to `None` and then it will be removed from the heat map. 

![image](https://user-images.githubusercontent.com/4793561/186008133-7d6c1331-604e-45f8-bb44-2434f0e827b0.png)



```python
from h2o_wave import main, app, Q, ui, data


@app('/')
async def serve(q: Q):

    heatmap_data_points = [
        ["C", "A", 0.5],
        ["C", "B", 0.75],
        ["C", "C", 1],
        ["B", "A", 0.25],
        ["B", "B", 1],
        ["B", "C", None],
        ["A", "A", 1],
        ["A", "B", None],
        ["A", "C", None],
    ]

    q.page['heatmap_plot'] = ui.plot_card(
        box='1 1 4 6',
        title='Heatmap Demo',
        data=data(fields=["y_value", "x_value", "value"], rows=heatmap_data_points),
        plot=ui.plot([
            ui.mark(
                type='polygon',
                x='=x_value',
                y='=y_value',
                color='=value',
                color_range='#ffffff green',
                label='=value'
            )
        ])
    )

    await q.page.save()
```
